pepper-apis/README.md

```diff
@@ -220,7 +216,7 @@ Note that the `RELOAD` priviledge is needed for some unit tests.
     If you don't do that, MySQL won't handle Unicode strings properly (one of the manifestations are '???' symbols in the
     MySQL CLI output of internationalized strings). Since we support multiple languages, this is a must.
 
-7. Update the _rendered_ secrets file `*.conf` (**not** `*.conf.ctmpl`), for example `output-build-config/testing-inmemorydb.conf`.
+7. Update the _rendered_ secrets file `*.conf` (**not** `*.conf.ctmpl`), for example `output-build-config/local.conf`.
```

Comment:
I understand the motivation to change the configuration file name (though it will be a bit of a minor pain to update all the shortcuts), but probably the new file should include the characters "test" somewhere (e.g.. test-local.conf or test.conf, etc) as that is the intent of the file.

Reply:
"test" as in "running unit tests"? I'm aware that folks use the file to run the server _locally_ as well outside the context of running tests.

Reply:
@yufengwng I'm onboard with the name change to `local.conf` (maybe `local-testing.conf`, but the `local` is the important context). Given how I've used that file in the past, it makes a great deal more sense.